meshapp/src/profile/profile-experience.tsx

```diff
@@ -0,0 +1,302 @@
+   * @param {string} props.description - Description of user's occupation
+   * @param {boolean} props.expand - Expand mode indicator for user's occupation
+   * @param {numberg} props.index- Index of user's occupation
+   */
+  const EditExperience = (props: {
+    title: string;
+    organization: string;
+    description: string;
+    expand: boolean;
+    index: number;
+  }) => {
+    return (
+      <Grid container alignItems="center" gap="10px">
+        <RemoveButton />
+        <Grid item {...(editMode ? { xs: 11.57 } : { xs: 12 })}>
+          <Accordion expanded={props.expand}>
+            <AccordionSummary
+              expandIcon={
+                <ExpandMoreIcon
+                  onClick={() => handleExpandClick(props.index)}
+                />
+              }
+            >
+              <Grid container gap="20px">
+                <Grid xs={5.8}>
+                  <ProfileTextField
+                    variant="standard"
+                    label="Occupation"
+                    placeholder=""
+                    text={props.title}
+                    charLimit={50}
+                    handleSave={(text) => saveOccupation(text, props.index)}
+                  />
+                </Grid>
+                <Grid xs={5.8}>
+                  <ProfileTextField
+                    variant="standard"
+                    label="Organization"
+                    placeholder=""
+                    text={props.organization}
+                    charLimit={50}
+                    handleSave={(text) => saveOrganization(text, props.index)}
+                  />
+                </Grid>
+              </Grid>
+            </AccordionSummary>
+            <AccordionDetails>
+              <ProfileTextField
+                variant="standard"
+                label="Description"
+                placeholder=""
+                text={props.description}
+                charLimit={50}
+                handleSave={(text) => saveDescription(text, props.index)}
+              />
+            </AccordionDetails>
+          </Accordion>
+        </Grid>
+      </Grid>
+    );
+  };
+
+  /**
+   * Displays the user's experience.
+   *
+   * @param props - Properties of the component
+   * @param {string} props.title - Title of user's occupation
+   * @param {string} props.organization - Name of business/org the user affiliates with
+   * @param {string} props.description - Description of user's occupation
+   * @param {numberg} props.index- Index of user's occupation
+   */
+  const ExperienceAccordion = (props: AccordionProps) => {
+    return (
+      <Grid item>
+        <Accordion>
+          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+            <Typography fontSize="18px" width="30%">
+              {props.title}
+            </Typography>
+            <Typography color="gray">{props.organization}</Typography>
+          </AccordionSummary>
+          <AccordionDetails>
```

Comment:
This is a duplicate of the `AccordionDetails` on line 198. Do we need both? If we do, I suggest making a constant or a function which provides this instead of writing it twice.

Reply:
Good point, I made it a constant 👍